tests/test_response.py

```diff
@@ -67,3 +69,7 @@ def test_response_body_passes_through_when_base64(self):
             "statusCode": 666,
             "isBase64Encoded": True,
         }
+
+    def test_unsported_header_raises(self) -> None:
+        with pytest.raises(RuntimeError):
+            Response(b"xxx", status_code=666)
```

Comment:
The name of the test is suggesting that the header is tested but here you put `body` and the `status_code`. So, what exactly is tested here? 🤔

Reply:
Also, the body expects to get `str` or `dict` but you provided `bytes` - `mypy` would catch that, I guess.